profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -479,6 +479,32 @@ metrics:
       - column:
           name: axVirtualServerPortStatName
           OID: 1.3.6.1.4.1.22610.2.4.3.4.4.1.1.4
+  # A table containing HA information.
+  - MIB: A10-AX-MIB
+    table:
+      name: axHAGroupStatusTable
+      OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2
+    symbols:
+      - name: axHAGroupStatusEntry
+        OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1
```

Comment:
the `*Entry` field in a MIB table is a placeholder for our needs and should not be part of this profile; please remove.